.travis.yml

```diff
@@ -29,7 +25,9 @@ matrix:
   fast_finish: true
 
 before_script:
-  - sudo /usr/share/elasticsearch/bin/plugin install -b mapper-attachments
+  - curl -O https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-5.0.0.deb
```

Comment:
Why have we locked elasticsearch version ? cannot be better to use a last 5.x stable?
https://www.elastic.co/guide/en/elasticsearch/reference/current/deb.html